src/xdocs/config_regexp.xml

```diff
@@ -585,7 +585,28 @@ public static final int A_SETPOINT = 1;
     value=&quot;System\.(out)|(err)\.print(ln)?\(&quot;/&gt;
 &lt;/module&gt;
         </source>
-
+        <p>
+          Example of violation from the above config:
```

Comment:
Just write: `Example:`
Documentation should be less verbose, and should clear the doubts of the person reading it with no extra words:)